src/core/webidl.js

```diff
@@ -753,9 +753,19 @@ function findDfn(defn, parent, name, definitionMap, type, idlElem) {
       name.replace(/[()]/g, "").replace(/\s/g, "-");
     dfn.id = id;
   }
-  dfn.dataset.idl = "";
+  dfn.dataset.idl = type === "enum-value" ? type : defn.type;
```

Comment:
Why `"enum-value"` has a special case?

Reply:
I think WebIDL2.js might (Not 💯 sure) be giving us the wrong type.

Reply:
Will check, and leave a comment to fix if it is.

Reply:
Confirmed. Webidl2.js has type “string” for enum values. Need to fix that, as its wrong.